README.md

```diff
@@ -90,17 +92,17 @@ This will spin up just the collection exercise service.
 
 ## Development
 
-Both RM and RH services can be started and stopped.
-The makefile contains targets that can start and stop eithr the RM services, RH services or both using:
+Both RM and RH services can be started and stopped. The makefile contains targets that can start and stop eithr the RM
```

Comment:
Spelling mistake in the word `either`. This is actually my fault for which I apologise